app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java

```diff
@@ -46,15 +51,20 @@ public void destroy() {
         if (refresh) {
             contributorsRepository.setDirty(true);
         }
-        return contributorsRepository.findAll().map(contributors -> {
-            setLoadingVisibility(View.GONE);
-            setRefreshing(false);
-            return Stream.of(contributors).map(contributor -> {
-                ContributorViewModel viewModel = new ContributorViewModel(contributor);
-                viewModel.setCallback(this);
-                return viewModel;
-            }).collect(Collectors.toList());
-        });
+        return contributorsRepository.findAll()
+                .map(contributors -> Stream.of(contributors).map(contributor -> {
+                    ContributorViewModel viewModel = new ContributorViewModel(contributor);
+                    viewModel.setCallback(this);
+                    return viewModel;
+                }).collect(Collectors.toList()))
+                .observeOn(AndroidSchedulers.mainThread())
```

Comment:
NOTE: Manipulation of `ObservableList` must be done in main thread. Otherwise, some events will not be notified.